CMakeLists.txt

```diff
@@ -439,7 +428,7 @@ set(XMP_INSTALLS)
 if(LIBXMP_BUILD_FULL AND BUILD_STATIC)
     add_library(xmp_static STATIC ${LIBXMP_SRC_FULL})
     list(APPEND XMP_INSTALLS xmp_static)
-    set_target_properties(xmp_static PROPERTIES OUTPUT_NAME xmp C_STANDARD 99)
+    set_target_properties(xmp_static PROPERTIES OUTPUT_NAME xmp C_STANDARD 90)
```

Comment:
Why not just lose the `C_STANDARD` thing altogether?

Reply:
For the record, I'm _not_ a professional coder :)

As for a contributor that can mistake, that's where the CI and the
reviewers come in to place.

So, I suggest that you lose it.

Reply:
> As for a contributor that can mistake, that's where the CI and the
reviewers come in to place.

The local compile failure is also important, that will reduce the major headache, so:
- All methods should be combined
- CI doesn't verify C90 builds, if they build with default sets, they use LATEST compilers that will use C99 and newer standards by default, so, no way to verify C90 compilation than manually
- Manual review while helps a lot, also a human factor, everybody with no exception may miss the small thing, but later it will led the compilation failure at real C90 compiler

Reply:
P.S. I started to enforce C90 compilation at C libraries because I had a lot of challenges with the code I had that contained a lot of mistakes made because of the knowledge lack, and locally it builds fine, on CI it builds fine, but it fails on the sudden place that I absolutely didn't expect. So, once I enabled the C90 enforcing, I resolved a huge doze of errors I hadn't resolved for a while, and later, I never got such errors as I had caught them quickly, and CI also does verify the C90 when the C90 is enforced by the build code.

Reply:
OK, I won't object any more.  (Others may have more to say.)